docs/accessibility.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: v2
+title: Accessibility
+bodyclass: a11y-statement-page
+---
+
+## Accessibility Statement for Leaflet
+
+This is an accessibility statement from Leaflet.
+
+<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" height="50px" viewBox="0 0 24 24" width="50px" style="fill:#199900;margin:-.75em 0 -1.5em 0;"><path d="M0 0h24v24H0z" fill="none"></path><path d="M20.5 6c-2.61.7-5.67 1-8.5 1s-5.89-.3-8.5-1L3 8c1.86.5 4 .83 6 1v13h2v-6h2v6h2V9c2-.17 4.14-.5 6-1l-.5-2zM12 6c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"></path></svg>
+
+### Conformance status
+
+The
+[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/)
+defines requirements for designers and developers to improve accessibility
+for people with disabilities.
+It defines three levels of conformance:
+Level A, Level AA, and Level AAA.
+
+Leaflet is partially conformant with WCAG 2.1
+[Level A](https://www.w3.org/TR/WCAG21/#cc1).
+Partially conformant means that some parts of the content do not fully conform
+to the accessibility standard.
+
+### Measures to support accessibility
+
+Leaflet strives to provide accessible maps.
+[Accessibility issues to be addressed](https://github.com/Leaflet/Leaflet/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility) are tracked and discussed
+on Github.
+
+### Feedback
+
+We welcome your feedback on the accessibility of Leaflet.
+If you encounter accessibility barriers, please check to see if there's a
+relevant bug or
+[file a new bug report](https://github.com/Leaflet/Leaflet/issues/new/choose).
+Or tweet
+[@LeafletJS](https://twitter.com/LeafletJS).
+
+
+### Compatibility with browsers and assistive technology
+
+Leaflet is designed to be compatible with the following assistive technologies:
+
+- Screen readers (partial compatibility)
+
+### Technical specifications
+
+Accessibility of Leaflet relies on the following technologies to work with
+the particular combination of web browser and any assistive technologies or
+plugins installed on your computer:
+
+- ARIA
+- CSS
+- HTML
+- JavaScript
+
+These technologies are relied upon for conformance with the accessibility
+standards used.
+
+### Limitations
+
+Despite our best efforts to ensure accessibility of Leaflet,
+there may be some limitations.
+
+Known limitations for Leaflet exist as
+[Github issues with the "accessibility" label](https://github.com/Leaflet/Leaflet/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility).
+
+### Evaluation report
+
+An evaluation report for Leaflet JS is available at:
+
+- [Web map tools WCAG 2.1 evaluation](https://github.com/Malvoz/web-maps-wcag-evaluation/blob/master/README.md)
```

Comment:
I'm the author of the linked Evaluation report. The template asked for an evaluation report, it even had an additional "Other evidence" heading which I excluded.

I'm fine with removing this, but if you want to keep it I'm fine with that too.

Reply:
(The linked 'accessibility' Github issues should be sufficient in providing any form of "evidence" or evaluation.)